tests/builder/app_settings.rs

```diff
@@ -263,6 +263,41 @@ fn infer_subcommands_pass_conflicting_aliases() {
     assert_eq!(m.subcommand_name(), Some("test"));
 }
 
+#[test]
+fn infer_long_flag_pass_conflicting_aliases() {
+    let m = Command::new("prog")
+        .infer_subcommands(true)
+        .subcommand(Command::new("test").long_flag_aliases(["testa", "t", "testb"]))
+        .try_get_matches_from(vec!["prog", "te"])
```

Comment:
This isn't testing long flags but command names

Reply:
Yeah, total brain fart here. Fixed all of them.